FirebaseAuth/Sources/Swift/SystemService/AuthAPNSToken.swift

```diff
@@ -30,13 +32,13 @@
     }
 
     /// The uppercase hexadecimal string form of the APNs token data.
-    lazy var string: String = {
+    var string: String {
```

Comment:
should this be a `let`?

Reply:
Computed properties are all `var`s.

Reply:
For future us– lazy vars aren't thread safe, so it needed to be computed and must then be a var. And, we can't do `let string: String = {...}()` pattern because it needs `self` for the `self.data` property.